plugin/pkg/shoot/oidc/clusteropenidconnectpreset/admission_test.go

```diff
@@ -191,13 +205,18 @@ var _ = Describe("Cluster OpenIDConfig Preset", func() {
 
 		})
 
-		Context("should mutate the result", func() {
+		Context("should mutate the result for shoot kubernetes <= 1.32", func() {
 			var (
-				expected *core.Shoot
+				expected     *core.Shoot
+				shootLess132 *core.Shoot // shoot version <1.32
 			)
 
 			BeforeEach(func() {
-				expected = shoot.DeepCopy()
+				shootNameLess132 := "shoot1"
+				shootLess132 = shoot.DeepCopy()
+				shootLess132.Name = shootNameLess132
+				shootLess132.Spec.Kubernetes.Version = "1.31.0"
+				expected = shootLess132.DeepCopy()
```

Comment:
Same as my suggestion in the other test:
```suggestion
				shoot.Spec.Kubernetes.Version = "1.31.0"
				expected = shoot.DeepCopy()
```

Reply:
hmm not sure about this in case tests would run in parallel. As i can see in each contexts BeforeEach we are doing deep copy probably because of this..